TrackerConditions/inc/StrawElectronics.hh

```diff
@@ -89,8 +89,7 @@ namespace mu2e {
       _TDCLSB(TDCLSB), _maxTDC(maxTDC), _TOTLSB(TOTLSB), _maxTOT(maxTOT), 
       _tdcResolution(tdcResolution), _electronicsTimeDelay(electronicsTimeDelay), 
       _ewMarkerROCJitter(ewMarkerROCJitter), 
-      _flashStart(flashStart), _flashEnd(flashEnd), 
-      _flashClockSpeed(flashClockSpeed),
+      _flashEnd(flashEnd), 
```

Comment:
Is flashend WRT the peak of the proton pulse time?

Reply:
flashend is a register set in electronics that sets the start of the digitization window wrt DAQ clock t=0